entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -219,6 +244,29 @@ Specifying `FromSql` anywhere other than on a `DbSet` had no added meaning or ad
 
 `FromSql` invocations should be moved to be directly on the `DbSet` to which they apply.
 
+<a name="notrackingresolution"></a>
+### No-tracking queries no longer perform identity resolution
+
+[Tracking Issue #13518](https://github.com/aspnet/EntityFrameworkCore/issues/13518)
+
+This change is introduced in EF Core 3.0-preview 6.
+
+**Old behavior**
+
+Before EF Core 3.0, the same entity instance would be used for every occurrence of an entity with a given type and ID. This matches the behavior of tracking queries.
+
+**New behavior**
+
+Starting with EF Core 3.0, different entity instances will be created when an entity with a given type and ID is encountered at different places in a query tree created using  `Include` statements. This matches the behavior of EF6.
```

Comment:
NIT: I am not sure that the mention of a query tree or `Include` helps here. IMO those are kind of orthogonal as an entity in the same "place" of the query tree is returned twice (e.g. for another "row" of the results) it will be a different instance. Also, this is not just for include but also for projected entities and collections. 

I think mentioning that if the same entity (i.e. same type and identity) is referenced multiple times in query results, then it will be a different instance each time. An example can help too, e.g. a query that returns Products and Categories will return a different Category for each product.

Reply:
Updated